### ISSUE TITLE
Does CONUS / OCONUS matter for entitlements?

### ISSUE THREAD

**Issue by aileendds** [deptofdefense/move.mil#152]
The 3rd dropdown on entitlements is CONUS vs OCONUS. However, this currently does nothing since we don't have any data in the db that corresponds to CONUS or OCONUS. Are we missing data or is this field unnecessary?

**Comment by lfantano** [comment]
We are missing data. It can be hidden right now because special item
instructions below is fixed.

With certain OCONUS bases those will change, Justin is working on compiling
that data for a v 2 later.

The V2 version would have a progressive form element where if the user
selects OCONUS they get an addition question to select the base.

**Comment by jde1984** [comment]
For the first pass at OCONUS restrictions - I have the Civilian folks but I removed them for this first pass since our form only asks service members right now.  But I think we can start with this... my thoughts would be after the member clicks OCONUS, it would prompt to ask them for their branch of service.   The locations that were applicable to them would appear in a drop-down to choose from and it would output the weight restriction message.  Right now, it's a generic message pulled from Appendix W.  
https://docs.google.com/a/dds.mil/spreadsheets/d/1jmlz7lA_9ZGw_El6lQ2O3wqJAFNONhDgv_RFyqhkOiA/edit?usp=sharing 

For the next iteration of the OCONUS entitlements, we have a few options...
1.  Similar to the above, but it breaks out both UB and HHG restrictions.  I started working on this part, but I wanted to get this first iteration out the door since I am way late with my blockers. OR; 
2. Go with the full monty - list ALL the OCONUS locations as part of the form questions and output what they are allowed as an HHG and UB.  For this, we need the list of POC's from Jill. 

Open to your thoughts designers at what functions best... (@lfantano @LillyMadigan )

**Comment by LillyMadigan** [comment]
@jde1984 Thanks for tracking down all this info!! Some questions; forgive me if they've already been addressed - and I can hop on a call today too :)

1. Are you asking for branch of service because only one branch has restrictions at a certain location? If an airman and a marine are both stationed in Okinawa, maybe only the marine has restricted weight allowances? 

OR is it that a marine base has across the board restrictions that an airbase doesn't? If it's possible for an Airman to be stationed OCONUS at a Marine base, then let's not limit the information the Airman has access to based on location. 

There are other ways to organize the list of locations so that it's not just one long overwhelming dropdown. A clickable map of the world could be one option - or a table organized by continental headers, OR by branch of service. I'd ask myself what the service member's mindset is coming to this page, and try to answer that question first. I met ppl who needed personalized content for their situation, and weren't interested in learning about all kinds of other move type. "I'm an E5 Soldier, with dependents, moving to Wiesbaden - what do I need to know?"

2. If we'd be misleading ppl by not breaking out UB and HHG, then I think we should go full monty. I wouldn't want someone to read the 1,000 lb limit for X location and make decisions based on 1/2 the story.

**Comment by jde1984** [comment]
@LillyMadigan - thanks for helping out! To answer the questions...
1. Yes... if you look at the JTR Appendix W, certain locations have restrictions based on which service the member belongs too.  It all comes back what the Services themselves will pay.  At the end of the day - it's all about the services and what they will fund/pay/etc. 

2. I don't think we are misleading folks, it will just take me a few more days to break it down from HHG and UB.  I wasn't sure how fast we wanted to push this out in terms of restrictions.

In terms of how we present the OCONUS piece... I love the map idea and I'd like to get inputs on ALL the OCONUS locations.  The list I have now are known areas with restrictions. But I'd love for a Service Member to click Germany and get all the details that way... it goes to the "customize your move" plan we had at the design-sprint.  That will take longer to assemble, so this may be a decision point for @Elliottdotgov and @lfantano - did we want to atleast get the restrictions out there now (right now we have a simple disclaimer) OR did we want to wait, get all the things about all the locations - knowing that may take some time to coord with all the JPPSO's.  

Thoughts?

**Comment by jillesmith1** [comment]
Team, adding USTC review--pls let me know if anyone has questions for us here.

**Comment by jde1984** [comment]
@jgarber623-gov @lfantano - I did a review of the content for all the OCONUS locations with restrictions. The list below highlights the locations with restrictions that have been called out. I believe this is all the content you needed, but let me know if you want me to break it out further. If I remember our call correctly, the intent was to display the restriction if they indicate they were going to one of those locations. 

https://docs.google.com/spreadsheets/d/1jmlz7lA_9ZGw_El6lQ2O3wqJAFNONhDgv_RFyqhkOiA/edit?usp=sharing

**Comment by lfantano** [comment]
Exactly. I think the UI changes we suggested was that if the user selected
OCONUS there would be a dropdown with the affected bases and an other. If
they select a base they will get a notification with their restriction.

Does this make sense @jgarber623-gov <https://github.com/jgarber623-gov>?
Or do you need a mock-up... :- /

**Comment by BP-DPS** [comment]
As of 24MAY2018: Transferred to Pivotal
